code/go/0chain.net/smartcontract/storagesc/allocation_pool.go

```diff
@@ -49,6 +49,28 @@ func (lr *lockRequest) decode(input []byte) (err error) {
 	return // ok
 }
 
+type readLockRequest struct {
```

Comment:
This is probably now in the wrong place, read pools are decoupled from allocation pools. I suggest moving to `readpool.go`.

Reply:
done